content/sips/sip-182.md

```diff
@@ -0,0 +1,161 @@
+This SIP makes it easier to deploy `Wrapper` contracts that can support more assets in the future and will help expand the total supply of synths.
+
+### Technical Specification
+<!--The technical specification should outline the public API of the changes proposed. That is, changes to any of the interfaces Synthetix currently exposes or the creations of new ones.-->
+
+#### WrapperFactory
+
+A new Synthetix core contract is created, `WrapperFactory`, which by means of the factory pattern deploys `Wrapper` instances using `createWrapper`. The `WrapperFactory` assumes functionality previously exposed by `EtherWrapper`.
+
+Specifically, `WrapperFactory` contains the following functions:
+- `createWrapper(IERC20 token, bytes32 currencyKey, bytes32 synthContractName)`
+  - Allows `owner` to create `Wrapper` contracts. `token` and `currencyKey` indicate which token and synth should be wrapped, and `synthContractName` is included because concatenation of `bytes32` strings within solidity is difficult, and should match the `AddressResolver` name of the contract for `currencyKey`.
+- `isWrapper(address possibleWrapper) returns (bool)`
+  - Allows Synthetix system components to verify a valid Wrapper contract by passing contract address. Returns boolean, indicating if its a Wrapper contract or not
+- `distributeFees()`
+  - Assumes functionality of `EtherWrapper.distributeFees`, sending fees to the fee pool and informing of status
+- `totalIssuedSynths() returns (uint)`
+  - TBD
+
+#### Wrapper
+
+The Wrapper created by the factory should be functionally equivalent to [SIP-112](sip-112.md), with the following exceptions:
+- Constructor accepts IERC20 token address `token` of source token, and currency key `currencyKey` of target synth
+- `sETHIssued` renamed to `synthIssued`
+- `maxETH` renamed to `maxTokenAmount`
+- `SystemSettings` values have additional `currencykey` argument to match
+- Upon `_mint` or `_burn`, equivalent value sUSD is minted into the `WrapperFactory` contract instead of withholding `sETH`
+- `distributeFees` removed
+
+`EtherWrapper` and `NativeEtherWrapper` are deprecated.
+
+#### DebtCache
+
+This class will be modified TBD
+
+#### SystemSettings
+
+External functions are added to match settings described in the `Configurable Values` section below.
+
+### Test Cases
+<!--Test cases for an implementation are mandatory for SIPs but can be included with the implementation..-->
+
+Wrapper initialized with WETH -> sETH
+- Given that a user has `u` amount of ETH and `u` WETH and the contract has `c` amount of ETH in spare capacity
```

Comment:
`u` and `u`? maybe `e` and `ew`?

I think you need an additional `Given` step to detail the creation of an ETH wrapper at address `A`, and then detail the creation of another wrapper say `LINK`. You've written these from the perspective of the user, but what about from the protocol that wants to create and manage a new wrapper?

cc @justinjmoses